api/src/main/scala/hmda/api/http/LarHttpApi.scala

```diff
@@ -66,6 +69,11 @@ trait LarHttpApi extends LarProtocol with ValidationResultProtocol {
       }
     }
 
+  def errorsAsResponse(list: List[String]): HttpResponse = {
+    val errorEntity = HttpEntity(ContentTypes.`application/json`, list.toJson.toString)
+    HttpResponse(StatusCodes.UnprocessableEntity, entity = errorEntity)
```

Comment:
I'll make the switch over to `400` (that does make more sense), but I think we should wait until we have a standard error format before building out the response, since we'll probably have to refactor it anyway once #433 is complete.